src/cli/cmd/farm/update/cmd.go

```diff
@@ -32,11 +33,15 @@ func NewCmd() *cobra.Command {
 		Short: "farm update updates the databse with latest farm release",
 		RunE:  cmdRunE,
 	}
+
+	cmd.Flags().StringVar(&flagDumpFilePath, "dumpFile", "", "use a pre-existing dump file for update instead of downloading fresh.")
```

Comment:
IMO we should keep a shorthand as well. may be `-f`

```suggestion
	cmd.Flags().StringVarP(&flagDumpFilePath, "dumpFile", "f", "", "use a pre-existing dump file for update instead of downloading fresh.")
```

Reply:
it's just that I wanted the user to be sure what they are doing i.e. I didn't want to make it easy (with a shorthand). my concern is - this tempers with the update logic in some ways and I have not put enough thought in this. once we are able to fit this reasonably into the update flow, we can then make it available to the users (unhide) and have a shorthand.